src/references/auth.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Authentication
+---
+
+## Authentication Sequence
+
+1. The User opens up a browser to check his cozy, using his *cozy app domain* (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...)
+2. he's redirected to his *cozy user domain* (`user.mycozy.cloud`)
+3. The browser request the *cozy user domain*, which send back the authentication form. The user fills his password, sending it back to the  *cozy user domain*
+4. The server *cozy user domain* set an auth cookie for *cozy user domain* ONLY
+5. A redirect status response code (302) for the requested app domain (`user-home.mycozy.cloud/`, or `user-drive.mycozy.cloud/`, or ...) is sent to the browser
+5. The browser request the *cozy app domain* with a [jwt code](https://jwt.io/)
+6. The jwt code  is checked and decoded through the stack, which send back a bearer token in the request header to *cozy app domain*
```

Comment:
The stack doesn't send a bearer token in the request header. Well, the stack doesn't sent the request here. And the JWT bearer token is sent in the body of the HTML response. It's then the client that extracts this token and send it to the stack on the *cozy user domain* as a bearer token in the Authorization header of HTTP requests.

Reply:
That was my mistake. Sorry Julie. I thought the code given to from the user domain to the app domain was a JWT and the bearer token was an simple unique string. It appears it's the opposite.

For the client, both string should be used as opaque strings anyway